scout/server/blueprints/omics_variants/views.py

```diff
@@ -97,7 +92,9 @@ def outliers(institute_id, case_name):
         case=case_obj,
         cytobands=cytobands,
         expand_search=get_expand_search(request.form),
-        filters=available_filters,
+        filters=set_persistent_filters_choices(
```

Comment:
Cool! Python is versatile: most languages might have had a closure, not allowing at least call-by-value modification of another argument to the same function in the same call, but call-by-name is more like call-by-reference in this sense I guess. 😸